### ISSUE TITLE
Do you have an ideas about Position Sensing?

### ISSUE THREAD

**Issue by mrunzo** [RyoKosaka/HelloDrum-arduino-Library#6]
I would like to implement positional sensing feature if you have any idea you can share an idea and resources.

**Comment by thijstriemstra** [comment]
Would love to see this feature as well.

**Comment by corrados** [comment]
Recently, I started researching the signal processing of a drum trigger module. As a first step I wanted to create a proof-of-conecpt where I ignore the latency and the processing requirements of the algorithms.

First results are shown here: https://github.com/corrados/mypitools/blob/master/projects/edrumtrigger/README.md.

I am not sure if this will lead to any useful algortihm which could be implemented on an Ardunio. But it is fun to do some research in that area :-).

**Comment by corrados** [comment]
I have now finished a first working version and created a short Youtube video to show my algorithms in action: https://youtu.be/6eQjCD-DFjo

**Comment by corrados** [comment]
I have ported the code to an ESP32 microcontroller now. The code is available at: https://github.com/corrados/edrumulus.
Here is a short video showing the trigger performance of the current implementation: https://youtu.be/naP-ODXl9Y0.

**Comment by thijstriemstra** [comment]
@corrados wow that's awesome! Have you tried it with any VST, eg. Superior Drummer? I will order a new ESP32 and make this a holiday project.

**Comment by corrados** [comment]
No, I have not done that yet. I have just used the Roland TDW-20 to generate the drum sound from the MIDI signal. If you want to try out my library, please keep in mind that the code is in a very early stage and only optimized for my PD-120 mesh pad. Also, you have to put the operating point of the ADC in the middle of its range (I do it with two resistors in series connection and a capacitor).

**Comment by thijstriemstra** [comment]
> the code is in a very early stage and only optimized for my PD-120 mesh pad. 

I am also using one of those PD-120 mesh pads 👍🏼

**Comment by SunboX** [comment]
Hey @mrunzo , I had the same in my mind. See issue #43 . I wrote the issue, before I saw yours.

**Comment by ojungleboy** [comment]
Very interested on this!
